### DISCUSSION TITLE
Webinar for french community

### DISCUSSION THREAD

**Discussion by pnavaro** [FourierFlows/FourierFlows.jl#363] [General]
Hi, I organize webinars for french community https://calcul.math.cnrs.fr/category/cafe.html . I'm looking for someone to talk about FourierFlows in 2024 ? 40 minutes + questions but It could be less.  It could be in english of course. Thanks in advance.

**Comment by navidcy** [comment]
I'd be keen to do this sometime during 2024! Perhaps from May onwards?

**Comment by pnavaro** [comment]
Great !  The webinar takes place once a month and May is for you. You can set the date now and keep the option of changing it a few weeks beforehand. It's usually at 10:00 AM, so it might be a bit late for you, I hope that's not a problem.

**Reply by pnavaro** [reply]
In May it is possible 6, 7, 13, 14, 16 or 17th

**Reply by navidcy** [reply]
I need to admit that at this point I'm having trouble figuring out how my schedule will look next month so scheduling something in May seems impossible! But pencil me down for one day and we can change later? I pick randomly May 16.

**Reply by pnavaro** [reply]
Sure no problem, since it is online, the talk could be cancelled or delayed. I set a reminder for me in march and contact you again. Thank you.

**Reply by navidcy** [reply]
Sounds great!

**Comment by pnavaro** [comment]
Dear @navidcy 
I set the date to May 16th but I noticed that you are in Melbourne. If a set the webinar at 10AM in Paris it will be 18PM in Melbourne. Is it ok for you ? We can still change the schedule. Here the link to our website https://calcul.math.cnrs.fr/cafe-julia-fourierflows.html
Pierre

**Reply by navidcy** [reply]
That's OK because I'll be in Cyprus then.
